pymedphys/_experimental/pinnacle/rtdose.py

```diff
@@ -201,10 +201,14 @@ def convert_dose(plan, export_path):
         ]
 
     # Read this from CT DCM if available?
-    if "HFS" in patient_position or "FFS" in patient_position:
-        ds.ImageOrientationPatient = [1.0, 0.0, 0.0, 0.0, 1.0, -0.0]
-    elif "HFP" in patient_position or "FFP" in patient_position:
-        ds.ImageOrientationPatient = [-1.0, 0.0, 0.0, 0.0, -1.0, -0.0]
+    if "HFS" in patient_position:
+        ds.ImageOrientationPatient = [1, 0, 0, 0, 1, 0]
+    elif "HFP" in patient_position:
+        ds.ImageOrientationPatient = [-1, 0, 0, 0, -1, 0]
+    elif "FFS" in patient_position:
+        ds.ImageOrientationPatient = [-1, 0, 0, 0, 1, 0]
+    elif "FFP" in patient_position:
+        ds.ImageOrientationPatient = [1, 0, 0, 0, -1, 0]
```

Comment:
Could we make a dictionary that looks like the following:

```python
image_orientation_map = {
	'FFDL': [0, 1, 0, 1, 0, 0],
	'FFDR': [0, -1, 0, -1, 0, 0],
	'FFP': [1, 0, 0, 0, -1, 0],
	'FFS': [-1, 0, 0, 0, 1, 0],
	'HFDL': [0, -1, 0, 1, 0, 0],
	'HFDR': [0, 1, 0, -1, 0, 0],
	'HFP': [-1, 0, 0, 0, -1, 0],
	'HFS': [1, 0, 0, 0, 1, 0]
}
```

From:

https://github.com/pymedphys/pymedphys/blob/40780daff9ccbf92dccc34bbc777207e0a409a72/pymedphys/_dicom/coords.py#L90-L101

And then, we can simply write:

```python
ds.ImageOrientationPatient = image_orientation_map[patient_position]
```

Given how ubiquitous that map is, it should probably sit within a new file placed at `pymedphys/_dicom/constants/orientation.py`.

And then all logic that uses this concept within PyMedPhys can import from there. For example the following function within PyMedPhys:

https://github.com/pymedphys/pymedphys/blob/40780daff9ccbf92dccc34bbc777207e0a409a72/pymedphys/_dicom/dose.py#L251-L256

Could be refactored to be:

```python
def require_patient_orientation(ds, patient_orientation):
    if not np.array_equal(
        ds.ImageOrientationPatient, 
        np.array(image_orientation_map[patient_position])
    ):
        raise ValueError(
            "The supplied dataset has a patient "
            f"orientation other than {patient_orientation}."
        )
```